### PR TITLE
feat - filter added

### PR DESCRIPTION
**Pull request by akash-1318**
Screenshot 2022-02-11 at 1 07 36 AM

Added filter component

### CODE REVIEW

File: component/filter/filter.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.filter__left{
```
Comment (tushargoyal812) [inline component/filter/filter.css:1]: use space before { its good for indentation,use beautify extension for this in vs code

**Review by tushargoyal812 — COMMENTED** [review]
looks good to merge just take what i suggested

File: component/filter/filter.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    text-decoration: underline;
+    text-underline-position: under;
+}
+
+.slider {
+    -webkit-appearance: none;
+    width: 100%;
+    height: 10px;
+    border-radius: 5px;
+    background: #d3d3d3;
```
Comment (arjunbharti) [inline component/filter/filter.css:37]: You can use color variable here

File: component/filter/filter.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    margin-top: 1rem;
+  }
+
+  .slider::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    appearance: none;
+    width: 1.5rem;
+    height: 1.5rem;
+    border-radius: 50%;
+    background: #636363;
```
Comment (arjunbharti) [inline component/filter/filter.css:51]: Color variable here too

**Review by arjunbharti — APPROVED** [review]
The UI is really nice and clean. Just added a little suggestion!

**Pull request merged**